swarm/storage/mru/resource.go

```diff
@@ -119,7 +118,8 @@ type LookupParams struct {
 	Limit   uint32
 }
 
-type resourceUpdate struct {
```

Comment:
Sorry for this change. It was a mistake. In the next step it will be renamed back.